MLS.Agent/CommandLine/VerifyCommand.cs

```diff
@@ -257,5 +271,28 @@ void SetOk()
                 }
             }
         }
+
+        private static bool ProjectIsCompatibleWithLanguage(string projectOrPackageName, string language)
```

Comment:
If this string really is a project or package name you shouldn't assume it's a file path...

Reply:
Maybe we want a custom type for this similar to the approach we discussed for `FileInfoOrUri`. Where did that land?

Reply:
"PackageSource"? I think...

Reply:
Just move that type to the right layer and rename I guess